README.md

```diff
@@ -1 +1,44 @@
-# Math-Magicians
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Math-Magicians
+
+> Description the project.
+
+> This is the first project of the Math Magicians application. I set up the environment and tools needed to develop a React application. In the following projects, I will develop the actual application.
+
+
+## Built With
+- GitFlow
+- React
+
+
+## Getting Started
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+- Web browser
+- Code editor
+
+### Setup
+```bash
+git clone (git@github.com:BouaikIlham/Math-Magicians.git)
+cd Math-Magicians
+```
```

Comment:
- [ ] Great job so far 👏🏻, however, It will be nice to add explicit steps to run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes   `npm install`, `npm start`, etc  🤗